spec/takeaway_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let (:item){ double :item }
+
+  it 'should have a function to view the menu' do
+    takeaway = Takeaway.new
+    expect(takeaway).to respond_to(:menu)
+  end
+
+  it 'should show you the menu when you call the function' do
+    takeaway = Takeaway.new
+    expect(takeaway.menu).to be_an_instance_of(Hash)
+  end
+
+  it 'should list a number of items and prices' do
+    takeaway = Takeaway.new
+    expect(takeaway.menu).not_to be_empty
+  end
+
+  it 'should show a number of items' do
+    takeaway = Takeaway.new
+    expect(takeaway.menu.length).to be > 5
+  end
+
+  it 'should have a function to order an item' do
+    takeaway = Takeaway.new
+    expect(takeaway).to respond_to(:order).with(1).argument
+  end
+
+  it 'should allow you to order an item x amount of times' do
+    takeaway = Takeaway.new
+    takeaway.order("pizza",2)
+    expect(takeaway.your_order.length).to eq 2
+  end
+
+  it 'should add items to your order when you order' do
+    takeaway = Takeaway.new
+    takeaway.order("pizza")
+    expect(takeaway.your_order).to include("pizza")
```

Comment:
Interesting failure from RSpec:

```
1) Takeaway should add items to your order when you order
     Failure/Error: expect(takeaway.your_order).to include("pizza")
       expected [["pizza", 10]] to include "pizza"
```

Why doesn't [["pizza", 10]] `include` "pizza"? What might `include` actually be doing, under the hood?